OSDOCS-17319 updates to ESO documentation (#102710)
openshift/openshift-docs: modules/external-secrets-operator-stablev1-channel.adoc

// Module included in the following assemblies:
//
// * security/external_secrets_operator/external-secrets-operator-install.adoc

:_mod-docs-content-type: CONCEPT
[id="external-secrets-operator-stablev1-channel_{context}"]
= About the {external-secrets-operator} stable-v1 channel

[role="_abstract"]

> Select the `stable-v1` channel to install and update the latest release of the {external-secrets-operator}. This channel helps ensure that you use the most recent stable release for your operator.

Reviewer: ```suggestion
Select the `stable-v1` channel to install and update the latest release of the {external-secrets-operator}. By selecting this channel, you can use the most recent stable release for your operator.
```
